bluepysnap/_plotting.py

```diff
@@ -36,7 +36,7 @@ def _get_pyplot():
             "Please pip install as follows:\n"
             "  pip install bluepysnap[plots] --upgrade"
         )
-        raise ImportError(str(e) + "\n\n" + msg)
+        raise ImportError(str(e) + "\n\n" + msg) from e
```

Comment:
this `str(e)` still needed?

Reply:
Good point, I think reporting the message with the available stack trace (w/ `from e`) is redundant. I'll remove it.